pom.xml

```diff
@@ -98,8 +101,13 @@ under the License.
     <dependency>
       <groupId>org.pac4j</groupId>
       <artifactId>pac4j-saml</artifactId>
-      <version>3.8.3</version>
+      <!-- versions 4.x.x require JDK 11 -->
```

Comment:
Then set `java.level` to `11` I guess? Not sure of status of https://github.com/jenkinsci/plugin-pom/pull/133.

Reply:
I think is fine, 3.9.0 was released a few days ago